server/eventlog.go

```diff
@@ -198,6 +199,10 @@ func getVerifiedCosState(coscel cel.CEL) (*pb.AttestedCosState, error) {
 			cosState.Container.OverriddenEnvVars[envName] = envVal
 		case cel.LaunchSeparatorType:
 			seenSeparator = true
+		case cel.MemoryMonitorType:
+			if len(cosTlv.EventContent) > 0 && cosTlv.EventContent[0] == uint8(1) {
```

Comment:
Can be more definitive here, by making sure the len(cosTlv.EventContent) == 1

Reply:
And for failsafe, I think the default value for cosState.HealthMonitoring.MemoryEnabled should be true, and when we see the eventContent == 0 here, we can turn it to false

Reply:
> Can be more definitive here, by making sure the len(cosTlv.EventContent) == 1

Done.

Reply:
> And for failsafe, I think the default value for cosState.HealthMonitoring.MemoryEnabled should be true, and when we see the eventContent == 0 here, we can turn it to false

Discussed offline, we agree to keep the default value as false.